backend/src/acidwatch_api/models/gibbs_minimization_model.py

```diff
@@ -108,7 +108,8 @@ async def run(self) -> RunResult:
         system.addComponent("CO2", co2_content, "mole/sec")
         for component, amount in self.concentrations.items():
             neqsim_name = self.formula_to_neqsim.get(component, component)
-            system.addComponent(neqsim_name, amount, "mole/sec")
```

Comment:
Is this change necessary for this PR to work? I'm thinking we shouldn't touch adapters unless strictly necessary.